skyvern/services/script_service.py

```
@@ -1954,20 +1954,24 @@ async def goto(
+            url=url,
+            label=block_validation_output.label,
+            output_parameter=block_validation_output.output_parameter,
+            parameters=block_validation_output.input_parameters,
+        )
+        await goto_url_block.execute_safe(
+            workflow_run_id=block_validation_output.workflow_run_id,
+            parent_workflow_run_block_id=block_validation_output.context.parent_workflow_run_block_id,
+            organization_id=block_validation_output.organization_id,
+            browser_session_id=block_validation_output.browser_session_id,
+        )
+    except Exception:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)